react-components/src/query/usePointCloudModelRevisionIdsFromReveal.test.tsx

```diff
@@ -69,24 +63,23 @@ describe(usePointCloudModelRevisionIdsFromReveal.name, async () => {
   });
 
   test('returns the relevant revision ids', async () => {
-    mockUse3dModels.mockReturnValue([
+    defaultDependencies.use3dModels.mockReturnValue([
       createPointCloudDMMock({
         revisionExternalId: TEST_REVISION_EXTERNAL_ID0,
         revisionSpace: TEST_SPACE
       })
     ]);
-    mockFdmSdkQueryNodesAndEdgesMock.mockResolvedValue(
-      createDmsPointcloudRevisionResponse(
-        TEST_MODEL_EXTERNAL_ID0,
-        TEST_REVISION_EXTERNAL_ID0,
-        TEST_SPACE
-      )
-    );
 
-    mockUseFdmSdk.mockReturnValue(
+    defaultDependencies.useFdmSdk.mockReturnValue(
       new Mock<FdmSDK>()
-        .setup((p) => p.queryNodesAndEdges<QueryRequest>)
-        .returns(mockFdmSdkQueryNodesAndEdgesMock)
+        .setup(async (p) => await p.queryNodesAndEdges<QueryRequest>(It.IsAny()))
+        .returnsAsync(
+          createDmsPointcloudRevisionResponse(
+            TEST_MODEL_EXTERNAL_ID0,
+            TEST_REVISION_EXTERNAL_ID0,
+            TEST_SPACE
+          )
```

Comment:
Just curious - is there a reason you prefer writing out the call to `queryNodesAndEdges` here, instead of leaving it as it was?

Reply:
I just found it more confusing to add one mock lib into another in this "simple" case. I found it easier to reason about after the change. Instead of a mock containing a mock where its implementation is defined somewhere else.